pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/generateNix.js

```diff
@@ -126,10 +126,25 @@ function fetchLockedDep(builtinFetchGit) {
       );
     }
 
+    if (resolved.startsWith("https://github.com/")) {
+      const s = resolved.split("/");
+
+      const [githubUrl, githubRev] = resolved.split("#");
+      const [, branch] = nameWithVersion.split("#");
+
+      return fetchgit(
+        fileName,
+        `${githubUrl}.git`,
+        githubRev,
+        branch || "master",
```

Comment:
```suggestion
        branch || "master",
```

Reply:
:+1: That should work fine and I guess the branch isn't even being used since we also pass a rev

Reply:
I also realized some time after commenting that branch probably won't even matter. Thank you for the change though!